package.json

```diff
@@ -31,29 +31,29 @@
   },
   "dependencies": {
     "@types/minimatch": "3.0.3",
-    "fs-extra": "^7.0.1",
+    "fs-extra": "^8.0.1",
     "handlebars": "^4.1.2",
-    "highlight.js": "^9.13.1",
+    "highlight.js": "^9.15.7",
     "lodash": "^4.17.11",
     "marked": "^0.6.2",
     "minimatch": "^3.0.0",
     "progress": "^2.0.3",
     "shelljs": "^0.8.3",
-    "typedoc-default-themes": "^0.6.0-0",
-    "typescript": "3.4.x"
+    "typedoc-default-themes": "file:../typedoc-default-themes/typedoc-default-themes-0.6.0-0.tgz",
```

Comment:
This probably isn't a good idea, if anyone working locally doesn't have the assumed file structure, it will break. If not npm, we should use a GitHub version https://docs.npmjs.com/files/package.json#github-urls

Reply:
This is also causing Travis to fail ^

Reply:
Yep, once I get a pre-release build of typedoc themes I'll update this